cmd/user-agent/src/main.js

```diff
@@ -32,17 +35,44 @@ async function loadAriesOnce() {
     return Vue.prototype.$aries
 }
 
-function ariesStartupOpts() {
-    // TODO start up option should be fetched from environment
+
+let defaultAriesStartupOpts = {
+    assetsPath: '/aries-framework-go/assets',
+    'outbound-transport': ['ws', 'http'],
+    'transport-return-route': 'all',
+    'http-resolver-url': [],
+    'agent-default-label': 'demo-user-agent',
+    'auto-accept': true,
+    'log-level': 'debug',
+    'db-namespace': 'agent'
+}
+
+async function ariesStartupOpts() {
+    let startupOpts = {}
+
+    if (process.env.NODE_ENV === "production") {
+        const axios = require('axios').default;
+
+        // call service to get the opts
+        await axios.get(window.location.origin + '/aries/jsopts')
+            .then(resp => {
+                startupOpts = resp.data
+                console.log("successfully fetched start up options: resp=" + JSON.stringify(startupOpts));
+            })
+            .catch(err => {
+                console.log("error fetching start up options - using default options : errMsg=", err);
+            })
+    }
+
     return {
-        assetsPath: "/aries-framework-go/assets",
-        "agent-default-label": "dem-js-agent",
-        "http-resolver-url": [],
-        "auto-accept": true,
-        "outbound-transport": ["ws", "http"],
-        "transport-return-route": "all",
-        "log-level": "debug",
-        "db-namespace": "agent"
+        assetsPath: defaultAriesStartupOpts['assetsPath'],
```

Comment:
so the defaults are the DEV options?

Reply:
yeah, plus which are not configurable externally - since path is part of Vue project.